.github/workflows/create-prerelease-branch.yml

```diff
@@ -0,0 +1,126 @@
+# Bump package version numbers and bundle changelogs into changelog file
+name: Create pre-release branch
+
+on:
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: 'Branch or tag to create pre-release from'
+        required: true
+        default: 'main'
+      bump_type:
+        description: 'Bump type'
+        required: true
+        type: choice
+        options:
+          - stable minor
```

Comment:
Can these be `stable-minor`, `stable-patch` and `beta` so that we get good branch names below?